src/pull.go

```diff
@@ -51,8 +52,47 @@ func (g *Commands) Pull(byId bool) error {
 	cl, clashes, err := pullLikeResolve(g, byId)
 
 	if len(clashes) >= 1 {
-		warnClashesPersist(g.log, clashes)
-		return ErrClashesDetected
+		if g.opts.FixClashes {
+			g.log.Logln("Some clashes found")
+			if g.opts.canPrompt() {
+				if !promptForChanges("Try fixing them [Y/N] ? ") {
+					return ErrClashesDetected
+				}
+			}
+			sort.Sort(ByPath(clashes))
+
+			lastPath := ""
```

Comment:
I'm working on this change (reworking your code slightly) and will push new version of this PR today. What would be the best place for autoRenameClashes function, should it still live in pull.go? Maybe move it to rename.go? or somewhere else? I didn't yet fully grasp the clashing problem in the project, maybe there are places other than pull that we should also take care off? Or maybe fixing clashes should be a separate command?